java/code/src/com/suse/manager/reactor/messaging/RegisterMinionEventMessageAction.java

```diff
@@ -340,6 +350,34 @@ private void applyMinionStartStates(String minionId, MinionServer registeredMini
             MinionList minionTarget = new MinionList(minionId);
             saltApi.updateSystemInfo(minionTarget);
         }
+        scheduleCoCoAttestation(registeredMinion);
```

Comment:
I don't have all the details, but are you sure we want to call attestation in both cases?

When in `saltbootInitrd`, then only saltboot state is executed and after it is finished, salt minion is automatically killed. Then initrd switch to system and we'll arrive here again only now outside `saltbootInitrd`.
So for saltboot terminal this means two attestations on one boot.

Reply:
Thanks for the info. Than it makes sense to only execute it outside of `saltbootInitrd`.